### ISSUE TITLE
1.9 Bot NPE

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#1143]
Seeing this on a bot at a startup
```
Fri Aug 26 01:04:17 EDT 2016 INFO [Triplea start thread] OUT -> Game Status: In Progress
Fri Aug 26 01:04:17 EDT 2016 SEVERE [Triplea start thread] ERROR -> Exception in thread "Triplea start thread" java.lang.NullPointerException
Fri Aug 26 01:04:17 EDT 2016 SEVERE [Triplea start thread] ERROR ->     at games.strategy.engine.framework.startup.launcher.ServerLauncher.launchInNewThread(ServerLauncher.java:171)
Fri Aug 26 01:04:17 EDT 2016 SEVERE [Triplea start thread] ERROR ->     at games.strategy.engine.framework.startup.launcher.AbstractLauncher.lambda$launch$97(AbstractLauncher.java:43)
Fri Aug 26 01:04:17 EDT 2016 SEVERE [Triplea start thread] ERROR ->     at java.lang.Thread.run(Thread.java:745)
```

**Comment by RoiEXLab** [comment]
@DanVanAtta This can be closed due to c9a8523 / #1144 rigth?

**Comment by DanVanAtta** [comment]
Yes, we actually then got another NPE, but it was because assets folder was missing. Which is a another bug in the error handling.
